lib/standalone.cjs

```diff
@@ -303,6 +304,15 @@ async function standalone({
  */
 function getFormatterFunction(selected) {
 	if (typeof selected === 'string') {
+		/** @see https://github.com/stylelint/stylelint/issues/7447 */
+		if (selected === 'github') {
```

Comment:
@marcalexiei
Thinking about it more, you will need to add `!quietDeprecationWarnings && ` to the condition, hence you should put that `if` where it was originally.
i.e. l88–94

---

@ybiquitous after this PR we should put the `if (!quietDeprecationWarnings)`inside the `emitDeprecationWarning` util.